.ci/validation.sh

```diff
@@ -1128,6 +1128,22 @@ ci-temp-check)
     exit ${#FILES_NO_CONCURRENCY[@]}
     ;;
 
+check-wildcards-on-pitest-target-classes)
+  ALL_CLASSES=$(xmlstarlet sel \
+    -N x=http://maven.apache.org/POM/4.0.0 \
+    -t -v "/x:project/x:profiles/x:profile//x:targetClasses/x:param" \
+    -n pom.xml)
+
+  CLASSES_NO_WILDCARD=$(echo "$ALL_CLASSES" | grep -v ".*\*\$" | grep -v -e '^\s*$' || echo)
+  CLASSES_NO_WILDCARD_COUNT=$(echo "$CLASSES_NO_WILDCARD" | grep -v -e '^\s*$' | wc -l)
```

Comment:
Strange that extra grep is required to make count.
CLASSES_NO_WILDCARD it might means that name is not correct.

Reply:
This avoids blank line that is result of previous grep.